.all-contributorsrc

```diff
@@ -54,6 +54,15 @@
       "contributions": [
         "financial"
       ]
+    },
+    {
+      "login": "MojixCoder",
+      "name": "Mojix Coder",
+      "avatar_url": "https://avatars.githubusercontent.com/u/76670309?v=4",
+      "profile": "https://github.com/MojixCoder",
+      "contributions": [
+        "maintenance"
+      ]
```

Comment:
@MojixCoder could you please delete this cause the bot automatically add it

Reply:
Done.